packages/@react-native-windows/cli/src/config/projectConfig.ts

```diff
@@ -99,6 +99,9 @@ export function projectConfigWindows(
   var validProject = false;
 
   if (usingManualOverride) {
```

Comment:
Why is this necessary? This blanket overwrites the logic for the last several lines of code.

Reply:
I see the problem, it's up in line 96 - I remember writing that, that's bad logic on my part.

that line should actually calculate the relative path from folder to sourceDir, then you won't need to set it again later.

Reply:
Thanks Jon, fixed it.